smoke-tests/spec/kubernetes_helper.rb

```diff
@@ -169,3 +169,14 @@ def get_nodes
 def get_daemonsets
   JSON.parse(`kubectl get daemonsets --all-namespaces -o json`).fetch("items")
 end
+
+# Scale replicas
+def scale_replicas(namespace, deployment, replicas = "")
```

Comment:
`replicas = ""` implies that you should supply a string value, but it makes more sense for this to be an integer.

As someone calling this method, I think it's reasonable to always require a value here, so I'd remove the default altogether.